From aa8fa0a4063a84564bb95ccf1c54b8d0b3aee338 Mon Sep 17 00:00:00 2001
From: Alexandre Torgue <alexandre.torgue@st.com>
Date: Mon, 7 Oct 2019 16:34:02 +0200
Subject: [PATCH 008/163] ARM: dts: stm32: fix regulator-sd_switch node on
 stm32mp157c-ed1 board

This commit fixes regulator-sd_switch node in order to be compliant to
DT validation schema.

Upstream-Status: Backport [2e7f46e13b3b29abeb201461535a529d0b8f3abc]
Signed-off-by: Alexandre Torgue <alexandre.torgue@st.com>
---
 arch/arm/boot/dts/stm32mp157c-ed1.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/stm32mp157c-ed1.dts b/arch/arm/boot/dts/stm32mp157c-ed1.dts
index 1d426ea8bdaf..329853d9b1de 100644
--- a/arch/arm/boot/dts/stm32mp157c-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157c-ed1.dts
@@ -100,7 +100,8 @@
 
 		gpios = <&gpiof 14 GPIO_ACTIVE_HIGH>;
 		gpios-states = <0>;
-		states = <1800000 0x1 2900000 0x0>;
+		states = <1800000 0x1>,
+			 <2900000 0x0>;
 	};
 };
 
-- 
2.27.0

